### PR TITLE
{chem}[intel/2019a] TAMkin v1.2.6 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3151.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/7895ba4fba79dea357d1e36419c2ef88 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2101.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6fc58ebc0ab86fee356639a36c34f6b7 for a full test report.

File: easybuild/easyconfigs/t/TAMkin/TAMkin-1.2.6-intel-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonPackage'
+
+name = 'TAMkin'
+version = '1.2.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://molmod.github.io/tamkin/'
```
Comment (Micket) [inline easybuild/easyconfigs/t/TAMkin/TAMkin-1.2.6-intel-2019a-Python-3.7.2.eb]: https

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/a948db0392338344be060fb17ce35187 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**